gluefactory/models/extractors/joint_point_line_extractor.py

```diff
@@ -362,16 +382,36 @@ def _forward(self, data):
 
             for df, af, kp, desc in zip(line_distance_field, line_angle_field, rescaled_kp, keypoint_descriptors):
                 line_data = {
-                    "points": torch.clone(kp),
```

Comment:
similar to previous comment we could think about maiing this configurable as well. I would also sugest adding print of a warning when deeplsd isn activatex so we see it when we accidentally have it running.